app/controllers/items_controller.rb

```diff
@@ -84,7 +84,7 @@ def show
 
   # display a flash message if the API response has an error
   def check_response
-    if !@res || (@res.key?("error") && @res.error)
+    if !@res || @res.error
```

Comment:
Same comment here as with what was done with the favicon commit regarding fixes made in other branches. This will actually conflict with what I improved upon the fix in the other branch now because it isn't the same. So you'll have to resolve the conflict and be careful to commit what was done in the other branch rather than this version.

*checks the PR page*

Oh no. Even worse, it's not even flagging it as a conflict. So if I hadn't looked at this closely, this branch would have been merged into `dev` and undone the fixes from the `error_checking` PR :weary:

Honestly, this is probably a good opportunity for you to learn interactive rebasing. That would be the way to make this branch/PR the cleaneast, but we can handle that in January. It would be a good lesson for everyone on the dev team :bulb:

Reply:
Scratch that. It wasn't showing the conflict yet because I hadn't merged the other PR back into `dev`. Now that I did that, it does show the conflict. Still, an interactive rebase would be the best way to remove the work done on the other branches from this one :+1:

Reply:
I realized this may be an issue, but it's hard to test code when it causes errors. Don't know if it was the case, but what if there were changes in the same file? It would be hard to leave that uncomitted.

Reply:
maybe I could rebase while I'm still working on the branch?

Reply:
An entire file's changes don't have to be committed. I use a git GUI program for line-by-line staging where I want some changes in a file committed and others not. I've never looked into how to do it from the command line, but it must be possible that way too. Just way easier with a GUI.

Now that the other branches have been merged, yeah it is a good time for rebasing. We'll go over it when everyone is back :+1:

Reply:
I attempted a rebase, removing the commits dealing with the favicon assets and the error checking. Hope I didn't mess anything up! It looks like there is no more merge conflict.

Reply:
I think the favicon commit is still there but looks like the error checking commit was removed in the rebase. Would still like to go through the process with Karin and Sarita to help them understand how it can be used.